account_payment_return_import_iso20022/wizard/camt_parser.py

```diff
@@ -143,7 +152,7 @@ def parse_payment_returns(self, ns, node):
 
     def check_version(self, ns, root):
```

Comment:
The error message "no camt.054.001.02:" do not take into account the compat with camt.053.

Reply:
Fixed